sdk/storage/azure-storage-blob/azure/storage/blob/_models.py

```diff
@@ -1296,6 +1136,140 @@ def __init__(self, **kwargs):
         self.rules = kwargs.pop('rules', None)
 
 
+class BlobProperties(DictMixin):
+    """Blob Properties."""
+
+    name: str
+    """The name of the blob."""
+    container: str
```

Comment:
This looks good except `container` here. Is this returned on GetBlobProperties and ListBlobs? If so, then we can go with it being required I guess.

Reply:
Yep, comes back on both:
![image](https://github.com/Azure/azure-sdk-for-python/assets/101599632/95c8f0a8-73e0-4549-8b5f-61b00bea8a58)